src/formatting/unit.rs

```diff
@@ -0,0 +1,81 @@
+use std::convert::TryInto;
+use std::fmt;
+
+use crate::errors::*;
+
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum Unit {
+    BitsPerSecond,
+    BytesPerSecond,
+    Percents,
+    Degrees,
+    Seconds,
+    Watts,
+    Hertz,
+    Bytes,
+    None,
+}
+
+impl fmt::Display for Unit {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(
+            f,
+            "{}",
+            match self {
+                Self::BitsPerSecond => "b/s",
+                Self::BytesPerSecond => "B/s",
+                Self::Percents => "%",
+                Self::Degrees => "°",
+                Self::Seconds => "s",
+                Self::Watts => "W",
+                Self::Hertz => "Hz",
+                Self::Bytes => "B",
+                Self::None => "",
+            }
+        )
+    }
+}
+
+impl TryInto<Unit> for &str {
+    type Error = crate::errors::Error;
+
+    fn try_into(self) -> Result<Unit> {
+        match self {
+            "b/s" => Ok(Unit::BitsPerSecond),
+            "B/s" => Ok(Unit::BytesPerSecond),
+            "%" => Ok(Unit::Percents),
+            "°" => Ok(Unit::Degrees),
```

Comment:
I don't think it makes much sense to use a degree symbol in configs. Should I change it to "degrees" or something?

Reply:
`deg` is the classig shorthand i think